Main.sublime-menu

```diff
@@ -208,14 +208,10 @@
         ]
     },
     {
-        "caption": "Preferences",
```

Comment:
I already see these changes in your first pull request.  Were you hoping for me to take this pull over the other one?